packages/common/src/ContractInteractor.ts

```diff
@@ -308,6 +309,55 @@ export class ContractInteractor {
     }
   }
 
+  async _validateERC165InterfacesRelay (): Promise<void> {
+    const allNull = [
+      this.penalizerInstance,
+      this.relayRegistrar,
+      this.relayHubInstance,
+      this.stakeManagerInstance].every(it => it == null)
+    if (allNull) {
+      throw new Error('ERC-165 interface check failed. Not a single contract instance initialized')
```

Comment:
1. very unreadable way to make a query. I actually understood what you're trying to do from the error message.
2. does a relay work with a partial list? AFAIK, it always require all entries, so why ever bother to support some?
3. (and if its only for testing, then this "allnull" check can simply be removed: it will check very soon that some are not null (sm, hub are checked from the worker)